docs/tutorials/basics/part2.md

```diff
@@ -21,24 +21,12 @@ Start the pipeline you created in Part 1 of the tutorial from a clean state:
 - If the pipeline is still running, click <icon icon="bx:stop-circle" /> to shutdown it down.
 - Click <icon icon="bx:play-circle" /> to restart the pipeline.
 
-Open the detailed view of the pipeline by clicking the <icon
-icon="bx:chevron-down" /> icon, copy the UUID of the pipeline and store it
-in an environment variable (this way you will be able to use
-`curl` commands in this tutorial verbatim):
-
-![Pipeline UUID](pipeline-uuid.png)
-
-```bash
-export PIPELINE_UUID=018a0ea6-9165-7e55-95f4-e5d865eeb272
-```
-
-
 ## Step 2. Subscribe to output changes
 
 Subscribe to changes to the `PREFERRED_VENDOR` view:
 
 ```bash
-curl -s -N -X 'POST' http://localhost:8080/v0/pipelines/${PIPELINE_UUID}/egress/PREFERRED_VENDOR?format=json | jq
+curl -s -N -X 'POST' http://localhost:8080/v0/pipelines/supply_chain/egress/PREFERRED_VENDOR?format=json | jq
```

Comment:
Maybe this was discussed, but this looks less secure.
In reality will there be some additional token you need to pass using curl?

Reply:
Yes, if you use authentication there will be an authorization header to pass.